core/math/bvh.h

```diff
@@ -282,15 +282,15 @@ class BVH_Manager {
 
 	// prefer calling this directly as type safe
 	void set_pairable(const BVHHandle &p_handle, bool p_pairable, uint32_t p_pairable_type, uint32_t p_pairable_mask) {
-		tree.item_set_pairable(p_handle, p_pairable, p_pairable_type, p_pairable_mask);
+		bool needs_update = tree.item_set_pairable(p_handle, p_pairable, p_pairable_type, p_pairable_mask);
```

Comment:
It might be useful to put a comment above saying something like:
`// returns true if the pairing state has changed and the item is active`
Just to make it clear what is happening without needing to read the sub function.